x-pack/plugins/osquery/cypress/e2e/all/alerts_cases.cy.ts

```diff
@@ -18,135 +18,121 @@ import {
     });
-
-    beforeEach(() => {
-      cy.login(ServerlessRoleName.SOC_MANAGER);
-      cy.visit('/app/security/rules');
-      clickRuleName(ruleName);
+    loadRule(true).then((data) => {
+      ruleId = data.id;
+      ruleName = data.name;
     });
+  });
 
-    after(() => {
-      cleanupPack(packId);
-      cleanupRule(ruleId);
-    });
+  beforeEach(() => {
+    loadRuleAlerts(ruleName);
+  });
 
-    describe('Case creation', () => {
-      let caseId: string;
+  after(() => {
+    cleanupPack(packId);
+    cleanupRule(ruleId);
+  });
 
-      before(() => {
-        interceptCaseId((id) => {
-          caseId = id;
-        });
-      });
+  describe('Case creation', () => {
+    let caseId: string;
 
-      after(() => {
-        cleanupCase(caseId);
+    before(() => {
+      interceptCaseId((id) => {
+        caseId = id;
       });
+    });
 
-      it('runs osquery against alert and creates a new case', () => {
-        const [caseName, caseDescription] = generateRandomStringName(2);
-        cy.getBySel('expand-event').first().click({ force: true });
-        cy.getBySel('take-action-dropdown-btn').click();
-        cy.getBySel('osquery-action-item').click();
-        cy.contains(/^\d+ agen(t|ts) selected/);
-        cy.contains('Run a set of queries in a pack').click();
-        cy.get(OSQUERY_FLYOUT_BODY_EDITOR).should('not.exist');
-        cy.getBySel('select-live-pack').click().type(`${packName}{downArrow}{enter}`);
-        submitQuery();
-        cy.get('[aria-label="Add to Case"]').first().click();
-        cy.getBySel('cases-table-add-case-filter-bar').click();
-        cy.getBySel('create-case-flyout').should('be.visible');
-        cy.getBySel('caseTitle').within(() => {
-          cy.getBySel('input').type(caseName);
-        });
-        cy.getBySel('caseDescription').within(() => {
-          cy.getBySel('euiMarkdownEditorTextArea').type(caseDescription);
-        });
-        cy.getBySel('create-case-submit').click();
-        cy.contains(`An alert was added to "${caseName}"`);
+    after(() => {
+      cleanupCase(caseId);
+    });
+
+    it('runs osquery against alert and creates a new case', () => {
+      const [caseName, caseDescription] = generateRandomStringName(2);
+      cy.getBySel('expand-event').first().click({ force: true });
+      cy.getBySel('take-action-dropdown-btn').click();
+      cy.getBySel('osquery-action-item').click();
+      cy.contains(/^\d+ agen(t|ts) selected/);
+      cy.contains('Run a set of queries in a pack').click();
+      cy.get(OSQUERY_FLYOUT_BODY_EDITOR).should('not.exist');
+      cy.getBySel('select-live-pack').click().type(`${packName}{downArrow}{enter}`);
+      submitQuery();
+      cy.get('[aria-label="Add to Case"]').first().click();
+      cy.getBySel('cases-table-add-case-filter-bar').click();
+      cy.getBySel('create-case-flyout').should('be.visible');
+      cy.getBySel('caseTitle').within(() => {
+        cy.getBySel('input').type(caseName);
       });
```

Comment:
```suggestion
      cy.getBySel('caseTitle').findBySel('input').type(caseName);
```

Reply:
done a bit different, findBySel didn't work in this case